codemods/jscodeshift/next/13/next-image-to-legacy-image/README.md

```diff
@@ -0,0 +1,33 @@
+This codemod safely migrates existing Next.js 10, 11, 12 applications importing `next/image` to the renamed `next/legacy/image` import in Next.js 13.

```

Comment:
by replacing `next/image` imports with `next/legacy/image` and replacing `next/future/image` imports with `next/image`